api/index.js

```diff
@@ -152,9 +132,7 @@ app.put('/proof', async(req, res) => {
     console.log(`got response: ${util.inspect(result)}`);
     res.json(result);
```

Comment:
please add 'return' before the res.json(result);

Reply:
I added return h&e: 
- app.get('/config', async (req, res)
- app.put('/proof', async(req, res)
- app.get('/key/:keyId', async (req, res)